torch/_dynamo/config.py

```diff
@@ -87,6 +87,9 @@
 # if an exception is encountered
 replay_record_enabled = False
 
+# Rewrite assert statement in python with torch._assert
+rewrite_assert_with_torch_assert = False
```

Comment:
I'd be ok turning this on by default.  It would also be good to turn it on and run CI to see if that shakes out any more issues.

Reply:
Ok let me flip it and run the CI

Reply:
Just did